contracts/contracts/VerificationGateway.sol

```diff
@@ -16,12 +17,13 @@ The gateway holds a single ProxyAdmin contract for all wallets, and can
 only called by a wallet that the VG created, and only if the first param
 is the calling wallet's address.
  */
-contract VerificationGateway
+contract VerificationGateway is EIP712
```

Comment:
@jacque006 Need to update docs/comments for VG to indicate it is already close to max byte size in current form.